manifests/03-role.yaml

```diff
@@ -78,6 +78,13 @@ rules:
   - "privileged"
   verbs:
   - use
+- apiGroups:
```

Comment:
@jcantrill does this need to go into the CSV?

Reply:
Correct.  This change does not affect the actual deployment and needs to go here for certain: https://github.com/openshift/cluster-logging-operator/blob/master/controller-manifests/cluster-logging.v4.1.0.clusterserviceversion.yaml and possibly here temporarily https://github.com/operator-framework/community-operators/blob/master/community-operators/cluster-logging/cluster-logging.v0.0.1.clusterserviceversion.yaml